DS change request: issue templates update (#2519)
cfpb/design-system: .github/ISSUE_TEMPLATE/component_change_or_add.md

---

> name: Component change or addtion request

Reviewer: ```suggestion
name: Component change or addition request
```

Author: fixed

about: Suggest an idea for this project
title: ''
labels: ''
assignees: ''

---

***For documentation requests, use the documentation request issue template instead.***
 

**Before filling out the information below, be sure you can answer "yes" to one of the following questions and be sure to include that detail in the issue:**

> ***Change to existing component:***

Reviewer: Meant to make this parallel with the wording of the other section title:
- Change an existing component
- Add a new component

> - Is the proposed change due to an accessbility or usability issue with the existing component?

Author: Fixed

- Does the change alter the existing component in only one major way (e.g. color change)? If so, consider if it should a variation of the existing component rather than a replacement.

***Adding a new component***
- Is the proposed component currently being used on more than one page type on [www.consumberfinance.gov](www.consumerfinance.gov)? Alternatively, can you provide examples of multiple places where the proposed component could be used?

> - Are existing Design System components not sufficiently able to server your product or users' needs, thus requiring the need to create a new component?

Reviewer 2: typo `able to server your`

- Is the proposed component different in more than one major way from existing components in the Design System? If the proposed component is different from an existing component in only one way (e.g. color), it would likely be better to add it as a variant of the existing component.

If you were able to answer "yes" to any of the above questions the next steps are to prepare your change request for review by adding the following information to the GitHub issue. 

> ***For component update or new addtions, please be sure to include the following***

Reviewer: ```suggestion
***For component updates or new additions, please be sure to include the following***
```

> - Indiccate if your change request is for making a change to an existing component of for adding a new one to the Design System

Reviewer: ```suggestion
- Indicate whether you are requesting to make a change to an existing component or requesting to add a new component to the Design System
```

- Add any data or research you have to show that the proposed change is accessbile and usable
- Supply any design and development context for your request you may have (e.g. Figma concepts, code samples, etc)

> - Include if you can design and build the proposed change or if you will need to request that someone makes the changes for you

Reviewer: ```suggestion
- Indicate whether you can design and build the proposed change or whether you will need to request that someone makes the changes for you
```

> - Draft documentation for the proposed change to be included in the Design System website (can be included in this issue or in the comments)

Reviewer: ```suggestion
- Provide documentation for the proposed change that will be included on the Design System webpage (this can be added here or as a comment)
```

Author: Updated but kep Draft since we already changed "Add any data ..." bullet point to "Provide any data ...". Don't want to repeat Provide ... in the same set of bullets.
